libsolidity/interface/CompilerStack.cpp

```diff
@@ -1322,7 +1322,7 @@ void CompilerStack::assemble(
 	}
 
 	// Throw a warning if EIP-170 limits are exceeded:
-	//   If contract creation returns data with length greater than 0x6000 (214 + 213) bytes,
+	//   If contract creation returns data with length greater than 0x6000 (2^14 + 2^13) bytes,
```

Comment:
Let me know which notation is preferred:
```suggestion
	//   If contract creation returns data with length greater than 0x6000 (2**14 + 2**13) bytes,
```